index.js

```diff
@@ -0,0 +1,244 @@
+const express = require("express");
+const helmet = require("helmet");
+const knex = require("knex");
+const dbConfigure = require("./knexfile");
+const db = knex(dbConfigure.development);
+const server = express();
+const port = 9000;
+
+server.use(helmet());
+server.use(express.json());
+
+//======== MIDDLEWARE =========//
+function verifyProject(req, res, next) {
+    const id = req.body.project_id;
+    db("projects")
+      .where({ id: id })
+      .then(project => {
+        if (project.length === 0) {
+          res.status(404).json({ message: "The project needs an ID" });
+        } else next();
+      })
+      .catch(err => res.status(500).json(err));
+  }
+
+//========== PROJECT ENDPOINTS ==========//
+server.get('/', (req, res) => {
+	res.send('<h2>Server is running.</h2>');
+})
+
+// GET PROJECTS
+server.get("/api/projects", (req, res) => {
+    db("projects")
+      .then(projects => {
+        res.status(200).json(projects);
+      })
+      .catch(err =>
+        res
+          .status(500)
+          .json({ error: "Uh oh! The project could not be retrieved." })
+      );
+  });
+
+  // POST
+  server.post("/api/projects", (req, res) => {
+    const project = req.body;
+    if (!project.name || !project.comments) {
```

Comment:
There is no 'comments' column in your migration for the projects table. Keep everything consistent.